test/Microsoft.SqlTools.ServiceLayer.UnitTests/ObjectExplorer/NodeTests.cs

```diff
@@ -91,55 +106,43 @@ public void ServerNodeConstructorShouldSetValuesCorrectly()
-                UserName = null
-            };
-            ConnectionCompleteParams connParams = new ConnectionCompleteParams()
-            {
-                ConnectionSummary = integratedAuthSummary,
-                ServerInfo = defaultServerInfo,
-                OwnerUri = defaultOwnerUri
-            };
+            oeServerInfo.UserName = null;
             // When querying label
-            string label = new ServerNode(connParams, ServiceProvider, serverConnection).Label;
+            string label = new ServerNode(oeServerInfo, serverConnection, ServiceProvider).Label;
             // Then only server name and version shown
-            string expectedLabel = defaultConnectionDetails.ServerName + " (SQL Server " + defaultServerInfo.ServerVersion + ")";
+            string expectedLabel = oeServerInfo.ServerName + " (SQL Server " + oeServerInfo.ServerVersion + ")";
             Assert.AreEqual(expectedLabel, label);
         }
 
         [Test]
         public void ServerNodeConstructorShouldShowDbNameForCloud()
         {
-            defaultServerInfo.IsCloud = true;
+            oeServerInfo.IsCloud = true;
 
             // Given a server node for a cloud DB, with master name
-            ServerNode node = new ServerNode(defaultConnParams, ServiceProvider, serverConnection);
+            ServerNode node = new ServerNode(oeServerInfo, serverConnection);
             // Then expect label to not include db name
-            string expectedLabel = defaultConnectionDetails.ServerName + " (SQL Server " + defaultServerInfo.ServerVersion + " - "
-                + defaultConnectionDetails.UserName + ")";
+            string expectedLabel = oeServerInfo.ServerName + " (SQL Server " + oeServerInfo.ServerVersion + " - "
+                + oeServerInfo.UserName + ")";
             Assert.AreEqual(expectedLabel, node.Label);
 
             // But given a server node for a cloud DB that's not master
-            defaultConnectionDetails.DatabaseName = "NotMaster";
-            defaultConnParams.ConnectionSummary.DatabaseName = defaultConnectionDetails.DatabaseName;
-            node = new ServerNode(defaultConnParams, ServiceProvider, serverConnection);
+            oeServerInfo.DatabaseName = "NotMaster";
+            node = new ServerNode(oeServerInfo, serverConnection, ServiceProvider);
 
             // Then expect label to include db name 
-            expectedLabel = defaultConnectionDetails.ServerName + " (SQL Server " + defaultServerInfo.ServerVersion + " - "
-                + defaultConnectionDetails.UserName + ", " + defaultConnectionDetails.DatabaseName + ")";
+            expectedLabel = oeServerInfo.ServerName + " (SQL Server " + oeServerInfo.ServerVersion + " - "
+                + oeServerInfo.UserName + ", " + oeServerInfo.DatabaseName + ")";
             Assert.AreEqual(expectedLabel, node.Label);
         }
 
         [Test]
         public void ToNodeInfoIncludeAllFields()

```

Comment:
Update test name if you're sticking with removing this method...(and search to make sure there aren't other things referencing this by name)